newIDE/app/src/EventsSheet/index.js

```diff
@@ -671,6 +679,27 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
       accelerator: 'CmdOrCtrl+Shift+Z',
     },
     { type: 'separator' },
+    {
+      label: i18n._(t`Collapse all`),
+      click: this.collapseAll,
+    },
+    {
+      label: i18n._(t`Expand all to level`),
+      submenu: [
+        {
+          label: i18n._(t`All`),
+          click: () => this.expandToLevel(-1),
+        },
+        { type: 'separator' },
+        ...[0, 1, 2, 3, 4, 5, 6, 7, 8].map(index => {
```

Comment:
I'd be interested to hear users' feedback on wether 0-8 is enough or not

Reply:
I remember seeing this 1 to 9 depth. The good thing is it matches the keyboard if we want to add shortcuts.